pkg/api/types.go

```diff
@@ -2239,6 +2238,12 @@ type PodAffinityTerm struct {
 	// Empty topologyKey is not allowed.
 	// +optional
 	TopologyKey string
+	// Selects Namespaces using cluster scoped-labels.  This
+	// matches namespaces selected by this label selector.
+	// This field follows standard label selector semantics.
+	// If present but empty, this selector selects all namespaces.
```

Comment:
Could you also add to the comment what is the behavior when this field is not present?